acvm-repo/brillig_vm/src/lib.rs

```diff
@@ -66,6 +69,10 @@ pub enum VMStatus<F> {
 // A sample for each opcode that was executed.
 pub type BrilligProfilingSamples = Vec<BrilligProfilingSample>;
 
+pub type Branch = (usize, usize);
```

Comment:
Could these two `usize` be documented? Alternative, maybe it could be a struct so the meaning is immediately clear.

Reply:
Added